echoregions/lines/lines.py

```diff
@@ -243,18 +236,17 @@ def filter_bottom(bottom, start_date, end_date):
                                 inside, outside."
                 )
 
-            # Interpolate on the sonar coordinates. Note that nearest interpolation has a problem
-            # when points are far from each other.
-            try:
-                bottom_interpolated = (
-                    filtered_bottom.reindex(joint_index)
-                    .interpolate(method=method, limit_area=limit_area)
-                    .loc[sonar_index]
-                ).fillna(max_depth)
-            except Exception as e:
-                # For all other such errors that may not necessarily deal with
-                # the specific values of method and limit_area.
-                print(e)
+            # Interpolate on the sonar coordinates. Note that nearest interpolation
+            # has a problem when points are far from each other.
```

Comment:
I think these 2 lines can also be added to a "Notes" section in the docstring to call attention to this.

Reply:
Just added them